docs/how-to/jupyter-notebook/create-notebook.rst

```diff
@@ -1,4 +1,52 @@
 Create Notebook
 ===============
 
-Create Notebook
\ No newline at end of file
+This guide provides instructions on how to create a Jupyter Notebook in the Data Science Stack (DSS) environment.
+
+Overview
+--------
+
+A Jupyter Notebook can be created using the DSS command line interface (CLI).  This notebook will include different packages and toolkits depending on the image used to create it.
+
+Prerequisites
+-------------
+
+Before creating a notebook, ensure you have the following:
+
+- DSS CLI installed on your workstation
+- DSS initialized
+
+Creating a Notebook
+-------------------
+
+1. **Select an image**:
+
+    Before creating a notebook, you need to select an image that includes the packages and toolkits you need.  To see a list of recommended images and their aliases, see:
+
+    .. code-block:: bash
+
+        dss list-images
+
+    For this guide, we will use the image `kubeflownotebookswg/jupyter-scipy:v1.8.0`
+
+2. **Create the notebook**:
```

Comment:
We can mention they can use the notebook image allias from `dss create --help``

Reply:
added it above